pkg/metrics/collector/sockstat_test.go

```diff
@@ -0,0 +1,76 @@
+/*
+Copyright 2018 Google Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package collector
+
+import (
+	"strings"
+	"testing"
+)
+
+func TestSockstats(t *testing.T) {
+	input := "sockets: used 270\n" +
+		"TCP: inuse 23 orphan 0 tw 10 alloc 52 mem 6\n" +
+		"UDP: inuse 3 mem 3\n" +
+		"UDPLITE: inuse 0\n" +
+		"RAW: inuse 0\n" +
+		"FRAG: inuse 0 memory 0"
+	input6 := "TCP6: inuse 14\n" +
+		"UDP6: inuse 1\n" +
+		"UDPLITE6: inuse 0\n" +
+		"RAW6: inuse 0\n" +
+		"FRAG6: inuse 0 memory 0"
```

Comment:
for mem, don't we also need to add in the RAW and FRAG memory?

Reply:
RAW doesn't have mem stats. FRAG memory is using a different unit with tcp/udp mem, and probably not common in GKE setup as well.

Reply:
Unchanged due to reason above.